src/Nethermind/Nethermind.JsonRpc/IJsonRpcConfig.cs

```diff
@@ -23,6 +23,19 @@ public interface IJsonRpcConfig : IConfig
             DefaultValue = "20000")]
         int Timeout { get; set; }
 
+        [ConfigItem(
+            Description = "The queued request limit for calls above the max concurrency amount for (" +
+                          nameof(IEthRpcModule.eth_call) + ", " +
+                          nameof(IEthRpcModule.eth_estimateGas) + ", " +
+                          nameof(IEthRpcModule.eth_getLogs) + ", " +
+                          nameof(IEthRpcModule.eth_newFilter) + ", " +
+                          nameof(IEthRpcModule.eth_newBlockFilter) + ", " +
+                          nameof(IEthRpcModule.eth_newPendingTransactionFilter) + ", " +
+                          nameof(IEthRpcModule.eth_uninstallFilter) + "). " +
+                          " If value is set to 0 limit won't be applied.",
+            DefaultValue = "500")]
+        int RequestQueueLimit { get; set; }
```

Comment:
@benaadams it is only for non-shared methods, shouldn't we mention it in the config name?

Reply:
Mentions the calls its for in the description?

User wouldn't understand what "non-shared" methods are; is an implementation detail?